README.md

```diff
@@ -52,6 +52,8 @@ generates:
 ```
 <!-- AUTO-GENERATED-CONTENT:END -->
 
+The `apollo-typed-documents` plugin also accepts the same `modulePathPrefix`, `relativeToCwd` and `prefix` config settings as [typescript-graphql-files-modules](https://graphql-code-generator.com/docs/plugins/typescript-graphql-files-modules).
```

Comment:
Could you move this to the top (after `The difference is that is uses...`).

I think it would fit there quite well, when already comparing with the original plugin.